internal/server/evaluation/evaluation.go

```diff
@@ -58,6 +60,74 @@ func (e *EvaluateServer) Variant(ctx context.Context, v *rpcEvaluation.Evaluatio
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(spanAttrs...)
 
-	e.logger.Debug("evaluate", zap.Stringer("response", resp))
+	s.logger.Debug("evaluate", zap.Stringer("response", resp))
 	return ver, nil
 }
+
+// Boolean evaluates a request for a boolean flag and entity.
+func (s *Server) Boolean(ctx context.Context, r *rpcEvaluation.EvaluationRequest) (*rpcEvaluation.BooleanEvaluationResponse, error) {
+	rollouts, err := s.store.GetEvaluationRollouts(ctx, r.FlagKey, r.NamespaceKey)
```

Comment:
i think we need to get the flag first for two reasons:
1. to make sure it exists, otherwise return a flag not found
2. ensure that the `type` on the flag is indeed `boolean`. same for the `variant` eval, just.. `variant` type